workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -123,11 +126,19 @@ protected virtual ReceptionistConfig GetReceptionistConfig(string workerType)
             }
             else
             {
+#if UNITY_ANDROID
```

Comment:
The function that gets the config is virtual to deal with cases like this. Please override rather than add conditional compilation to the base class.